CMakeLists.txt

```diff
@@ -606,15 +606,21 @@ if (BUILD_TESTING)
             set(FUZZ_TIMEOUT_SEC 60)
         endif()
 
-        if(NOT DEFINED CORPUS_UPLOAD_LOC)
+        if(DEFINED ENV{CORPUS_UPLOAD_LOC})
+            set(CORPUS_UPLOAD_LOC $ENV{CORPUS_UPLOAD_LOC})
+        else()
             set(CORPUS_UPLOAD_LOC "none")
         endif()
 
-        if(NOT DEFINED ARTIFACT_UPLOAD_LOC)
+        if(DEFINED ENV{ARTIFACT_UPLOAD_LOC})
+            set(ARTIFACT_UPLOAD_LOC $ENV{ARTIFACT_UPLOAD_LOC})
+        else()
             set(ARTIFACT_UPLOAD_LOC "none")
         endif()
 
-        if(NOT DEFINED FUZZ_TESTS)
+        if(DEFINED ENV{FUZZ_TESTS})
+            set(FUZZ_TESTS $ENV{FUZZ_TESTS})
+        else()
             set(FUZZ_TESTS "${TESTS}")
         endif()
```

Comment:
Can you describe what this block of code is doing, and why? What would happen if you removed it?

Reply:
I figured this variable isn't being used. I removed it and both FuzzBatch and FuzzBatchScheduled runs fine without it. I linked & updated the successful CodeBuild job in the PR description. Thank you for pointing it out!